src/core/primitives/RovingFocusGroup/fragments/RovingFocusRoot.tsx

```
@@ -0,0 +1,18 @@
+import React, { Children } from 'react';
+
+
+
+export interface RovingFocusRootProps {
+    children: any
+}
+
+function RovingFocusRoot({children} : RovingFocusRootProps) {
```

ESLint

[error] A space is required after '{'.

(object-curly-spacing)